src/async/Services/GraphQL.js

```diff
@@ -0,0 +1,99 @@
+const { GraphQLClient } = require("graphql-request");
+const {
+  ExternalZenatonError,
+  InternalZenatonError,
+  ZenatonError,
+} = require("../../Errors");
+const { credentials } = require("../../client");
+
+function getError(err) {
+  // Validation errors
+  if (err.response && err.response.errors && err.response.errors.length > 0) {
+    const message = err.response.errors
+      .map((graphqlError) => {
+        const path = graphqlError.path ? `(${graphqlError.path}) ` : "";
+        const errorMessage = graphqlError.message
+          ? graphqlError.message
+          : "Unknown error";
```

Comment:
a little JS trick:
You can write here `const errorMessage = graphqlError.message || "Unknown error"`